### PR TITLE
MEN-7198: prepare repository

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: interfaces/v1/rootfs-image [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        $mender_cli commit
+        ;;
+
+    Provides)
+        $mender_cli show-provides
+        # Also submit device_type, which is in a separate file, not in the database. We don't know if the
+        # file contains a newline, so it's important that this is done last, to avoid corrupting other
+        # entries.
+        grep '^device_type=' /var/lib/mender/device_type
+        ;;
```
Comment (lluiscampos) [inline interfaces/v1/rootfs-image:54]: @kacf I copied this file as-is from the `mender-update-orchestrator` repository. Should be add `Inventory` API?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+UPDATE_INTERFACES_V1_ROOTFS_IMAGE = interfaces/v1/rootfs-image
+
+UPDATE_INTERFACES_V1_ALL := $(shell find interfaces/v1 -type f)
+
+
+all:
+
+install: \
+	install-interfaces
```
Comment (kacf) [inline Makefile:14]: I found this line breaking a bit confusing, maybe it's better to keep it on the same line for readability?

One more below as well.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+install-interfaces: $(UPDATE_INTERFACES_V1_ALL)
+	install -m 755 -d $(prefix)$(datadir)/interfaces/v1
+	install -m 755 $^ $(prefix)$(datadir)/interfaces/v1/
+
+install-interface-rootfs-image:
```
Comment (kacf) [inline Makefile:20]: There is a way in makefiles to make dynamic targets for each update interface I think, but I can't remember it off the top of my head. Maybe @vpodzime knows?

File: README.md [hunk 4]
```diff
@@ -1 +1,49 @@
-# mender-orchestrator-update-interfaces
\ No newline at end of file
+Mender Orchestrator Update Interfaces
+=====================================
+
+Mender is an open source over-the-air (OTA) software updater for embedded Linux devices. Mender comprises a client running at the embedded device, as well as a server that manages deployments across many devices.
+
+This repository contains community supported Update Modules. An Update Module is an extension to the Mender client for supporting a new type of software update, such as a package manager, container, bootloader or even updates of nearby microcontrollers. An Update Module can be tailored to a specific device or environment (e.g. update a proprietary bootloader), or be more general-purpose (e.g. install a set of .deb packages.).
```
Comment (kacf) [inline README.md:6]: The README needs a lot of polishing, both this paragraph and below.

File: interfaces/v1/rootfs-image (hunk 1 above)
Comment (kacf) [reply interfaces/v1/rootfs-image:54]: Yes, we should!

File: README.md (hunk 4 above)
Comment (lluiscampos) [reply README.md:6]: :see_no_evil:

File: Makefile (hunk 3 above)
Comment (lluiscampos) [reply Makefile:20]: I know what to do it :muscle:

Comment (kacf) [reply Makefile]: Nice!

File: README.md [hunk 5]
```diff
@@ -1 +1,52 @@
...
+Orchestrator is a Mender Enterprise component responsible for updating the devices in a system.
+
+![Mender logo](https://github.com/mendersoftware/mender/raw/master/mender_logo.png)
+
+## Getting started
+
+To start using Mender, we recommend that you begin with the Getting started section in [the Mender
+documentation](https://docs.mender.io/).
+
+To start using Mender Update Modules, we recommend that you begin with the Update Modules -
```
Comment (kacf) [inline README.md:18]: Still referring to Update Modules. I guess we don't really have a section on Interfaces yet, but maybe you can add a point to fix this in [MEN-7016](https://northerntech.atlassian.net/browse/MEN-7016), once we get to it?

**Review by kacf — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1 +1,52 @@
...
+
+We welcome and ask for your contribution. If you would like to contribute to Mender, please read our
+guide on how to best get started [contributing code or
+documentation](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md).
+
+## License
+
+All files in this repository are licensed under the Apache License, Version 2.0, unless indicated
+otherwise.
+[LICENSE](https://github.com/mendersoftware/mender-orchestrator-update-interfaces/blob/master/LICENSE)
```
Comment (vpodzime) [inline README.md:34]: *See LICENSE...*?

**Review by vpodzime — COMMENTED** [review]
I'm not sure what exactly to review here.

**Comment by lluiscampos** [comment]
> I'm not sure what exactly to review here.

@vpodzime Kristian ping-ed you for the `Makefile` suggestion, but I went ahead and fixed it as I understood what he meant. Otherwise not really much for you, but thanks for reading through it :eyes:

**Comment by vpodzime** [comment]
> I went ahead and fixed it as I understood what he meant

That explains why I couldn't see them :grin: :+1:

**Pull request merged**